src/includes/getting-started-config/javascript.nextjs.mdx

```diff
@@ -1,14 +1,17 @@
-```Npx
+Though it's possible to [configure everything manually](/platforms/javascript/guides/nextjs/manual-setup/), we have a wizard which will automate the initial steps. To use it, run the following from the root level of your project:
+
+```bash
 npx @sentry/wizard -i nextjs
 ```
 
-<Note>
+The wizard will prompt you to log into Sentry, and then will automatically add the necessary configuration files to your project. It will perform the following changes:
+
+- create `sentry.client.config.js` and `sentry.server.config.js` with the default `Sentry.init`
+- create `next.config.js` with the default configuration
+- create `sentry.properties` with configuration for `sentry-cli` (which is used when automatically uploading source maps)
 
-The call to the [Sentry Wizard](https://github.com/getsentry/sentry-wizard) will automatically patch your project with the necessary configuration, though you can [setup manually](/platforms/javascript/guides/nextjs/manual-setup/) if you prefer.
-It will perform the following changes:
+See the docs on [manual configuration](/platforms/javascript/guides/nextjs/manual-setup/) for examples of the files created by the wizard. If any of these files already exists, it will be created with a leading underscore and you will need to merge it with the existing file manually.
```

Comment:
```suggestion
See [manual configuration](/platforms/javascript/guides/nextjs/manual-setup/) for examples of the files created by the wizard. If any of these files already exists, it will be created with a leading underscore and you will need to merge it with the existing file manually.
```